### PR TITLE
KS-SK1035-Update-403_forbidden-for-WebHosting-TRANSLATIONS

### PR DESCRIPTION
**Pull request by Kazarus4**
All translations are done and links updated.

Need proof for DE and EN.
Need proof too  for Latin and PL versions

### CODE REVIEW

File: pages/web/hosting/diagnostic_403_forbidden/guide.es-es.md [hunk 1]
```diff
@@ -1,106 +1,142 @@
 ---
 title: "¿Qué hacer en caso de página 403 forbidden?"
-excerpt: Cómo volver a publicar un sitio web cuando este muestre la página 403 forbidden
+excerpt: Descruba cómo volver a publicar un sitio web cuando este muestre la página 403 forbidden
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_403_forbidden/guide.es-es.md]:
```suggestion
excerpt: Descubra cómo volver a publicar un sitio web cuando este muestre la página 403 forbidden

```

File: pages/web/hosting/diagnostic_403_forbidden/guide.es-es.md [hunk 2]
```diff
@@ -1,106 +1,142 @@
...
 #### Reactivar el alojamiento con FileZilla
 
 > [!primary]
 >
-> Si desea instalar **FileZilla** para manipular los archivos de su sitio web, siga las indicaciones de esta [guía](../web_hosting_guia_de_uso_de_filezilla/).
+> Para instalar el software **FileZilla** para manipular los archivos de su sitio web, siga las instrucciones de este [tutorial](/pages/web/hosting/ftp_filezilla_user_guide).
 >
 
-Abra FileZilla y conéctese al espacio de almacenamiento. En el menú superior, haga clic en `Servidor`{.action} y seleccione `Introducir comando personalizado`{.action} (el nombre puede variar ligeramente según la versión de FileZilla que utilice).
+Abra FileZilla y luego [conéctese a su espacio de almacenamiento FTP](/pages/web/hosting/ftp_connectio). Haga clic en `Servidor`{.action} en el menú y, seguidamente, en `Introducir un pedido FTP`{.action} (el título puede ser diferente según la versión de FileZilla que utilice).
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_403_forbidden/guide.es-es.md]:
```suggestion
Abra FileZilla y luego [conéctese a su espacio de almacenamiento FTP](/pages/web/hosting/ftp_connection). Haga clic en `Servidor`{.action} en el menú y, seguidamente, en `Introducir un pedido FTP`{.action} (el título puede ser diferente según la versión de FileZilla que utilice).

```

File: pages/web/hosting/diagnostic_403_forbidden/guide.es-es.md [hunk 3]
```diff
@@ -114,30 +150,58 @@ SITE CHMOD 705 /
...
+```
+SITE CHMOD 705 .
+```
+
 #### Reactivar el alojamiento con el explorador FTP "net2ftp"
 
 En el [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es), acceda a la sección `Web Cloud`{.action} y, seguidamente, haga clic en `Alojamientos`{.action} en la pestaña `FTP-SSH`{.action} del alojamiento correspondiente.
 
-Haga clic en el botón `Explorador FTP `{.action} y conéctese al espacio de almacenamiento siguiendo las instrucciones de esta [guía](../conexion-espacio-almacenamiento-ftp-alojamiento-web/#21-conexion-mediante-un-explorador-ftp). Haga clic en el botón `Avanzado`{.action} y, a continuación, en el botón `Go`{.action} situado junto a "Send arbitrary FTP commands to the FTP server".
+Pulse el botón `Explorador FTP `{ y conéctese a su [espacio de almacenamiento FTP](/pages/web/hosting/ftp_connection).
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_403_forbidden/guide.es-es.md]: why the { ?

File: pages/web/hosting/diagnostic_403_forbidden/guide.es-es.md [hunk 4]
```diff
@@ -114,30 +150,58 @@ SITE CHMOD 705 /
...
 ![net2ftp](images/net2ftp.png){.thumbnail}
 
-Introduzca el siguiente comando en la parte superior de la página y haga clic en el botón con forma de "V" verde.
+Introduzca el siguiente comando en la parte superior de la página:
+
+```
+SITE CHMOD 705 /
+```
+
+y haga clic en el botón con forma de "V" verde.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_403_forbidden/guide.es-es.md]:
```suggestion
Haga clic en el botón con forma de "V" verde.

```

File: pages/web/hosting/diagnostic_403_forbidden/guide.es-us.md [hunk 5]
```diff
@@ -1,106 +1,142 @@
 ---
 title: "¿Qué hacer en caso de página 403 forbidden?"
-excerpt: Cómo volver a publicar un sitio web cuando este muestre la página 403 forbidden
+excerpt: Descruba cómo volver a publicar un sitio web cuando este muestre la página 403 forbidden
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_403_forbidden/guide.es-us.md]:
```suggestion
excerpt: Descubra cómo volver a publicar un sitio web cuando este muestre la página 403 forbidden

```

File: pages/web/hosting/diagnostic_403_forbidden/guide.it-it.md [hunk 6]
```diff
@@ -110,30 +152,59 @@ SITE CHMOD 705 /
...
 ![net2ftp](images/net2ftp.png){.thumbnail}
 
-Nella parte superiore della pagina, inserisci il comando qui sotto e clicca su "V" verde.
+Nella parte superiore della pagina, inserisci questo comando:
+
+```
+SITE CHMOD 705 /
+```
+
+e clicca su "V" verde.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_403_forbidden/guide.it-it.md]:
```suggestion
Clicca su "V" verde.

```

File: pages/web/hosting/diagnostic_403_forbidden/guide.pl-pl.md [hunk 7]
```diff
@@ -114,30 +152,58 @@ SITE CHMOD 705 /
...
 ![net2ftp](images/net2ftp.png){.thumbnail}
 
-W górnej części strony wprowadź poniższe polecenie, a następnie kliknij zielony przycisk "V".
+W górnej części strony wprowadź poniższe polecenie:
+
+```
+SITE CHMOD 705 /
+```
+
+następnie kliknij zielony przycisk "V".
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_403_forbidden/guide.pl-pl.md]:
```suggestion
Następnie kliknij zielony przycisk "V".

```

File: pages/web/hosting/diagnostic_403_forbidden/guide.pt-pt.md [hunk 8]
```diff
@@ -1,90 +1,128 @@
 ---
 title: O que fazer em caso de página "403 forbidden"?
-excerpt: Saiba como repor o seu site online quando apresenta uma página "403 forbidden"
+excerpt: Descruba como repor o seu site online quando apresenta uma página "403 forbidden"
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_403_forbidden/guide.pt-pt.md]:
```suggestion
excerpt: Saiba como repor o seu site online quando apresenta uma página "403 forbidden"

```

File: pages/web/hosting/diagnostic_403_forbidden/guide.pt-pt.md [hunk 9]
```diff
@@ -114,30 +152,58 @@ SITE CHMOD 705 /
...
 ![net2ftp](images/net2ftp.png){.thumbnail}
 
-Na parte superior da página, introduza o comando abaixo e clique no botão que representa um "V" verde.
+Na parte superior da página, insira o seguinte comando:
+
+```
+SITE CHMOD 705 /
+```
+
+e clique no botão que representa um "V" verde.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_403_forbidden/guide.pt-pt.md]:
```suggestion
Clique no botão que representa um "V" verde.

```

File: pages/web/hosting/diagnostic_403_forbidden/guide.es-es.md (hunk 3 above)
Comment (Kazarus4) [reply pages/web/hosting/diagnostic_403_forbidden/guide.es-es.md]: DONE

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**